polaris-react/src/components/DataTable/DataTable.scss

```diff
@@ -300,28 +298,22 @@
     flex-direction: column;
     width: 100%;
     overflow: hidden;
+    border-spacing: 0;
```

Comment:
I couldn't make out that this made any difference to the final output, but I might have not been testing the correct scenario.

Reply:
Maybe you weren't scrolled down so the sticky header was not visible? I double checked and it's definitely needed to remove the extra spacing on the sticky header. I had to add it because I changed the markup of the sticky header to `<table>` to match everything else.

Reply:
Makes sense. Thank you.